pyo3-build-config/src/impl_.rs

```diff
@@ -1391,13 +1391,11 @@ fn default_cross_compile(cross_compile_config: &CrossCompileConfig) -> Result<In
         .unwrap_or(PythonImplementation::CPython);
 
     let lib_name = if cross_compile_config.target.operating_system == OperatingSystem::Windows {
-        let mingw = cross_compile_config.target.environment == Environment::Gnu;
-
         Some(default_lib_name_windows(
             version,
             implementation,
             abi3,
-            mingw,
+            false,
```

Comment:
I think this is correct, because the other call site gets the `mingw` flag value from the Python interpreter itself, not from the current Rust compile target, like I did above:

https://github.com/PyO3/pyo3/blob/ea37acb8e2b01084ee9bec90e01979ce65d51161/pyo3-build-config/src/impl_.rs#L258-L265